README.md

```diff
@@ -48,15 +48,16 @@ Below is an example on how to use `FirebaseAuth` with a redirect upon sign-in:
 // Import FirebaseAuth and firebase.
 import React from 'react';
 import StyledFirebaseAuth from 'react-firebaseui/StyledFirebaseAuth';
-import firebase from 'firebase';
+import { initializeApp } from 'firebase/app';
+import { getAuth, EmailAuthProvider, GoogleAuthProvider } from 'firebase/auth';
 
 // Configure Firebase.
 const config = {
   apiKey: 'AIzaSyAeue-AsYu76MMQlTOM-KlbYBlusW9c1FM',
   authDomain: 'myproject-1234.firebaseapp.com',
   // ...
 };
-firebase.initializeApp(config);
+const firebaseApp = initializeApp(firebaseConfig);
```

Comment:
Don't see where the `firebaseConfig` is coming from. I think you missed its import.

Reply:
Thanks, updated README.md.

Reply:
Great, thanks!